client/state/themes/actions/activate.js

```diff
@@ -29,33 +22,9 @@ export function activate(
 	siteId,
 	source = 'unknown',
 	purchased = false,
-	keepCurrentHomepage = false
+	keepCurrentHomepage = true
```

Comment:
I'm setting `keepCurrentHomepage: true` as default here, but it looks too fragile. We can also set `dontChangeHomepage: true` as default in the action `activateTheme`: https://github.com/Automattic/wp-calypso/blob/7bca130291db0f8a0af6cb7262bc4cdbedeca99f/client/state/themes/actions/activate-theme.js#L40 

What do you think?